typescript/test/test-application/test-ts.ts

```diff
@@ -67,18 +68,18 @@ describe('Test application for TypeScript users', () => {
     this.timeout(240000);
     // copy protos to generated client library and copy test application to local.
     if (!fs.existsSync(path.join(showcaseLib, 'protos'))) {
-      fs.copySync(protos, path.join(showcaseLib, 'protos'));
+      await copy(protos, path.join(showcaseLib, 'protos'));
```

Comment:
Maybe these tests could check the existence of the files after copying?

Reply:
Oh. So the thing is, in the recent commit, I removed "test applications" because they were just too flaky and were actually broken for some time. So this code is gone completely.